src/components/StateChannelsPanel.vue

```diff
@@ -47,6 +48,7 @@ const loadNextStateChannels = () => {
 
 const loadStateChannels = () => {
   fetchStateChannels({ limit: limit.value })
+  fetchStateChannelsCount()
```

Comment:
I guess this could be fetched on the server-side but it's not a big deal. I'm fine with leaving it as it is

Reply:
Yep, let's keep it consistent. FIxed